app/models/enterprise.rb

```diff
@@ -127,6 +127,9 @@ class Enterprise < ApplicationRecord
               message: Spree.t('errors.messages.invalid_instagram_url')
             }, allow_blank: true
   validate :validate_white_label_logo_link
+  validates :external_billing_id,
```

Comment:
I think this commit belongs to another pull request.

Reply:
Yep, sorry about that, just fix it.